config/districts.yml

```diff
@@ -1,2 +1,30 @@
-- Eixample
-- Ciutat Vella
+- 
+  - Ciutat Vella
```

Comment:
Actually I found :boar: pretty cute. Like hairy and giant :pig: BUT I implemented your approach because I think it's easier to make changes.